src/reference/informative/informA1.rst

```diff
@@ -8,4 +8,32 @@
 
   .. container:: infospec
 
-    **TODO** Informal notes go here
+    .. container:: heading3
+
+      Namespaces
+
+    Namespaces are a way of making sure that names and definitions are interpreted within the right frame of reference.  In CellML, two namespaces are used.  These are the CellML namespace itself (which helps to define the :code:`units` elements as distinct from the XML default units system) and the MathML namespace used to interpret the :code:`<math>` elements.  For more information on namespaces in general, please refer to the :namespace_help:`W3 schools namespace page <>`.  For more information on the MathML namespace please refer to the :mathml2help:`W3 MathML namespace <#inferf.namespace>` page.
```

Comment:
- One sentence per line (same everywhere else).
- "which[, for example], helps..."
- Replace ":code:`<math>`" with ":code:`math`".
- Rather than having "For more information..." twice, might want to merge the two sentences.

Reply:
ef89774

Reply:
ef89774 hasn't be pushed, I guess...?

Reply:
Not yet :)